src/Rakaposhi.Core/Rakaposhi.UnitTest/UserStatusTest.cs

```diff
@@ -65,5 +95,140 @@ public void UserStatusCreate_InValid()
             Assert.IsTrue(condition: ErrorCode.ADDERROR == response.Value.ToString());
```

Comment:
@MariamShaikh When you comparing values must have to use areEqual. IsTrue only use for asserting Boolean values.

Reply:
fixed